### ISSUE TITLE
Add exercise solutions

### ISSUE THREAD

**Issue by ErinBecker** [datacarpentry/organization-genomics#91]
[Episode 2](https://datacarpentry.org/organization-genomics/02-project-planning/index.html) and [episode 3](https://datacarpentry.org/organization-genomics/03-ncbi-sra/index.html) are missing episode solutions.

**Comment by hoytpr** [comment]
We should be able to close #91 after #103. However if the answers need tweaking we can open a new issue.
